src/openrct2/ride/Vehicle.h

```diff
@@ -516,8 +516,8 @@ enum
     SOUND_RANGE_NONE = 255
 };
 
-#define VEHICLE_SEAT_PAIR_FLAG 0x80
-#define VEHICLE_SEAT_NUM_MASK 0x7F
+constexpr uint8_t kVehicleSeatPairFlag{ 0x80 };
```

Comment:
```suggestion
constexpr uint8_t kVehicleSeatPairFlag = 0x80;
```

(Applies to all of them.)

Reply:
You are right, I will make sure to replace those...